src/state-summary/state-card-climate.js

```diff
@@ -30,12 +30,11 @@ class StateCardClimate extends PolymerElement {
 
   static get stateInfoTemplate() {
     return html`
-    <state-info 
-      hass="[[hass]]" 
-      state-obj="[[stateObj]]" 
-      in-dialog="[[inDialog]]" 
-      override-name="[[overrideName]]">
-    </state-info>
+    <state-info
+      hass="[[hass]]"
+      state-obj="[[stateObj]]"
+      in-dialog="[[inDialog]]"
```

Comment:
In retrospect, `in-dialog` is such a misnomer as attribute for `state-info`. It defines context instead of behavior (`show-last-changed`) while state info should not know anything about dialogs. Oh well, so we learn